velox/exec/tests/SpillTest.cpp

```diff
@@ -156,16 +156,19 @@ class SpillTest : public ::testing::TestWithParam<common::CompressionKind>,
     // the batch number of the vector in the partition. When read back, both
     // partitions produce an ascending sequence of integers without gaps.
     spillStats_.wlock()->reset();
+    common::PrefixSortConfig prefixSortConfig = common::PrefixSortConfig();
```

Comment:
Let's add one more parameterized test to cover with prefix sort enable and disabled? Thanks!